spec/mastermind_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'mastermind'
+
+
+game = Mastermind.new
```

Comment:
this is putting the instance on the scope of the file. The "risk" in using this for each tests is that you might end up with false failing, or even false passing tests. The ideal situation is to have a clean working state for each test example so that state from a previous test won't pollute the state of the current test. Rspec has a couple of helpers (e.g. [before](https://relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks) and [let](https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let)) you can use to facilitate that while still keeping your code DRY.